ibm/resource_ibm_cis_edge_functions_action.go

```diff
@@ -0,0 +1,187 @@
+package ibm
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	cisEdgeFunctionsActionScriptName = "script_name"
+	cisEdgeFunctionsActionScript     = "script"
+	cisEdgeFunctionsActionScriptFile = "/tmp/script.js"
+)
+
+func resourceIBMCISEdgeFunctionsAction() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMCISEdgeFunctionsActionCreate,
+		Read:     resourceIBMCISEdgeFunctionsActionRead,
+		Update:   resourceIBMCISEdgeFunctionsActionUpdate,
+		Delete:   resourceIBMCISEdgeFunctionsActionDelete,
+		Exists:   resourceIBMCISEdgeFunctionsActionExists,
+		Importer: &schema.ResourceImporter{},
+		Schema: map[string]*schema.Schema{
+			cisID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "CIS Intance CRN",
+			},
+			cisDomainID: {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "CIS Domain ID",
+				DiffSuppressFunc: suppressDataDiff,
+			},
+			cisEdgeFunctionsActionScriptName: {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
Name cannot be updated.. Please add 
forceNew to the name

Reply:
done